### ISSUE TITLE
Add a lower priority than normal.

### ISSUE THREAD

**Issue by roachsinai** [mpcjanssen/simpletask-android#970]
For now, there are priorities from A to Z and no priority (normal), so I think it will be better if there is a `:WAIT` priority lower than normal.

**Comment by mpcjanssen** [comment]
Not supported by the todo.txt format.  But what is wrong with using no priority as priority normal?

**Comment by roachsinai** [comment]
Like I have a idea which I will do it in future, for now I just want to record it. I think give it a `:WAIT` is come naturally.

**Comment by mpcjanssen** [comment]
You can already use tags or lists for that. Not sure how a special priority will help.

**Comment by roachsinai** [comment]
But, I want it list under the todos with normal priority, is there a way to make it?

**Comment by vojtechkral** [comment]
Feels to me that this is basically about sorting only. If I understand correctly, you'd like _tasks_ without priority to sort somewhere in between tasks _with_ priority. Say, for example, that a task without a priority would sort priority-wise the same as a task with priority of `M`, which is about in the middle of the spectrum. Then all tasks with priority lower than that would sort at the bottom and vice versa. Would that cover your requirement?

I don't think there's a direct support for this in the UI, but you can probably do this with Lua.

**Comment by mpcjanssen** [comment]
With Lua pretty much any sort can be done.

**Comment by roachsinai** [comment]
@vojtechkral yes, it is about sorting only. But is not tasks without priority to sort somewhere in between tasks with priority. As now all priority is higher than no priority, even `Z` is in higher priority than `-`.

The request is why not have a priority lower than `-`?

> onSort (task, fields, extensions) -> string
>Called for every task as part of sorting the todo list. This function should return a string for every task. This string is then used to sort the tasks.
>
>Returns
> The string to use for task sorting.

I'm a little confused about `onSort`, what should I return in bleow code?

```
funtion onSort(t, f, e)
   if string.find(f.text, ":WATI") then
       return 
   end
end
```

**Comment by mpcjanssen** [comment]
What is wrong with treating no priority as wait? Anyway if for tasks with :WAIT you return 2 and els return 1 all :WAIT tasks will be sorted last.

**Comment by mpcjanssen** [comment]
Also you can use t instead of f.text.

**Comment by smichel17** [comment]
I'd use f.text, it's much easier to modify scripts that use it later.

**Comment by roachsinai** [comment]
> You should define the onSort function in the filter (not in the configuration). Defining it in the main configuration will not work, if the Filter script is empty, the onGroup function will be undefined.

![webwxgetmsgimg](https://user-images.githubusercontent.com/9500049/56373405-70648200-6233-11e9-80c6-82fa415c90a7.jpeg)

But my setting same as screenshot above not worked. Could you please tell me where is wrong?

**Comment by roachsinai** [comment]
> What is wrong with treating no priority as wait? 

Hi @mpcjanssen as maybe lua script can resolve my problem, proves nothing is wrong.

But for your question, everything is fine maybe cause you think every task should has high priority than normal. For me, a task could have priority lower than normal.

**Comment by mpcjanssen** [comment]
You also need to move the Lua script to the top in the sort tab.

**Comment by roachsinai** [comment]
@mpcjanssen Thanks for your instruction! Worked.

**Comment by roachsinai** [comment]
Seems Lua script not work at first when run this app, you have to point something, like settings (just point setting then return the script will run).
